packages/slice-machine/server/src/api/state.ts

```diff
@@ -108,6 +108,15 @@ export default async function handler(
     ...frontEnv,
     sliceMachineAPIUrl: baseUrl,
     shortId: prismicData.shortId,
+    updateVersionInfo: {
```

Comment:
I think this should be removed, it's only for your local test right ?

Reply:
whoops, I didn't mean to commit that :/